cli/cmd/plugin/ui/web/tanzu-ui/src/views/unmanaged-cluster/unmanaged-cluster-common/UnmanagedClusterCard/UnmanagedClusterCard.scss

```diff
@@ -1,3 +1,7 @@
 label {
     color: var(--cds-global-color-gray-700);
 }
+
+.unknown {
```

Comment:
recommendation: change css class name to .status-unknown

Reply:
Done

Reply:
Done